apps/aecontract/src/aect_contracts.erl

```diff
@@ -145,14 +148,21 @@ is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_2, abi := ?ABI_AEVM
         ?ROMA_PROTOCOL_VSN    -> false;
         ?MINERVA_PROTOCOL_VSN -> true;
         ?FORTUNA_PROTOCOL_VSN -> true;
-        ?LIMA_PROTOCOL_VSN    -> false %% TODO: Revise this before release
+        ?LIMA_PROTOCOL_VSN    -> false
     end;
 is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_3, abi := ?ABI_AEVM_SOPHIA_1}, ProtocolVersion) ->
     case ProtocolVersion of
         ?ROMA_PROTOCOL_VSN    -> false;
         ?MINERVA_PROTOCOL_VSN -> false;
         ?FORTUNA_PROTOCOL_VSN -> true;
-        ?LIMA_PROTOCOL_VSN    -> true %% TODO: If you bump to VM_AEVM_SOPHIA_3 please turn this off!
+        ?LIMA_PROTOCOL_VSN    -> false
+    end;
+is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_4, abi := ?ABI_AEVM_SOPHIA_1}, ProtocolVersion) ->
+    case ProtocolVersion of
+        ?ROMA_PROTOCOL_VSN    -> false;
+        ?MINERVA_PROTOCOL_VSN -> false;
+        ?FORTUNA_PROTOCOL_VSN -> false;
+        ?LIMA_PROTOCOL_VSN    -> true %% TODO: If you bump to VM_AEVM_SOPHIA_4 please turn this off!
```

Comment:
The comment is symmetric to the previous code, but I am not sure what it means?

Reply:
It i a bit cryptic, I agree. It was originally intended as a reminder to turn off creation of contracts with old VM:s but it doesn't read well.

Reply:
Shouldn't it at least say `SOPHIA_5` then?